### PR TITLE
docs: add uv run docs for gui scripts

### PR DESCRIPTION
**Pull request by samypr100**
## Summary

Follow up docs to https://github.com/astral-sh/uv/pull/6453

### CODE REVIEW

**Comment by samypr100** [comment]
Here's a rendered view

rendered _docs

File: docs/guides/scripts.md [hunk 1]
```diff
@@ -212,6 +212,54 @@ $ uv run --python 3.10 example.py
...
+
+widget.setLayout(grid)
+widget.setGeometry(100, 100, 200, 50)
+widget.setWindowTitle("uv")
+widget.show()
+sys.exit(app.exec_())
+```
+
+```console
+PS> uv run --with "PyQt5<6" example_pyqt.pyw
```
Comment (zanieb) [inline docs/guides/scripts.md:258]: Why `<6`?

File: docs/guides/scripts.md [hunk 2]
```diff
@@ -212,6 +212,54 @@ $ uv run --python 3.10 example.py
...
+
+```console
+PS> uv run example.pyw
+```
+
+![Run Result](../assets/uv_gui_script_hello_world.png){: style="height:50px;width:150px"}
+
+Similarly, it works with dependencies as well:
+
+```python title="example_pyqt.pyw"
```
Comment (zanieb) [inline docs/guides/scripts.md:238]: Should we use inline dependency metadata here?

Comment (zanieb) [reply docs/guides/scripts.md:238]: I guess it's a bit easier to read with the `--with`.

**Review by zanieb — APPROVED** [review]
Started to suggest this should go in the concept document and realized I didn't write one yet :( 

Thank you for following up!

File: docs/guides/scripts.md (hunk 1 above)
Comment (samypr100) [reply docs/guides/scripts.md]: Old habit, removed in a679d7a41282e2a4425b21bcd6feb127a5178355

File: docs/guides/scripts.md (hunk 2 above)
Comment (samypr100) [reply docs/guides/scripts.md:238]: Yup

**Pull request merged**